### ISSUE TITLE
ArrayRenderer throws exception in case the referenced array property is missing

### ISSUE THREAD

**Issue by edgarmueller** [eclipsesource/jsonforms#180]
```
TypeError: Cannot read property 'length' of undefined
    at ArrayRenderer.render (http://localhost:8123/app/bower_components/jsonforms/dist/js/jsonforms.js:1068:91)
    at RenderService.render (http://localhost:8123/app/bower_components/jsonforms/dist/js/jsonforms.js:635:42)
    at http://localhost:8123/app/bower_components/jsonforms/dist/js/jsonforms.js:651:38
```

**Comment by koegel** [comment]
fix by following #199

**Comment by eneufeld** [comment]
@edgarmueller  Can this still happen with the latest refactoring?
